active_storage_base64.gemspec

```diff
@@ -1,13 +1,14 @@
 Gem::Specification.new do |s|
   s.name = 'active_storage_base64'
-  s.version = '0.0.0'
+  s.version = '1.0.0'
```

Comment:
Following https://semver.org/#how-should-i-deal-with-revisions-in-the-0yz-initial-development-phase maybe we can start with `0.1.0` until we have some apps in prod using the gem?